bin/liveness_probe

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env -S ruby --disable=gems
+
+$file_path = ENV.fetch("RACECAR_LIVENESS_PROBE_FILE_PATH", "")
+$liveness_max_interval = Integer(ENV.fetch("RACECAR_LIVENESS_PROBE_MAX_INTERVAL")) rescue nil
+
+file_path_and_max_interval_set? or exit(1)
+liveness_file_exists? or exit(1)
+liveness_within_timeout_interval? or exit(1)
+
+# Didn't exit so all is well
+$stderr.puts("Racecar healthcheck OK #{elapased_since_liveness_event}")
+exit(0)
+
+
+BEGIN {
```

Comment:
Would moving the method definitions to the top achieve the same result as having `BEGIN` here? I'm not even sure what it does...

Reply:
Yes, `BEGIN` changes the flow of the file so you can put the method calls at the top. Allows the reader to see the 'what' first without the 'how' getting in the way, but I'm not wedded to it.

Reply:
Back in the old days I'd just use a `main` method that I would call from the bottom of the file. But I think probably we'll want to call into this code from `racecarctl` anyway.

Reply:
Oh same, but then I got old and weird 😆